tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "noUnusedLocals": true,
     "jsx": "react",
     "jsxFactory": "h",
-    "types": ["lodash-es", "d3"]
+    "types": ["lodash-es", "d3", "jest"]
```

Comment:
I'd like not to add jest types to the main tsconfig.json. It looks like it's possible to define a custom tsconfig for jest (that extends the global one) by modifying jest config like so:
```
{
  ...
  "globals": {
    "ts-jest": {"tsConfigFile": "tsconfig.test.json"}
  }
}
```